### PR TITLE
fix: clean up CI issues for new DataContext

### PR DESCRIPTION
**Pull request by ImCesar**
A fork of https://github.com/cypress-io/cypress/pull/18211

Continuing to fix CI issues

- [x] Have tests been added/updated?
- [ ] Has the original issue or this PR been tagged with a release in ZenHub? 
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

**Review by tgriesser — APPROVED** [review]

File: packages/frontend-shared/src/graphql/testUrqlClient.ts [hunk 1]
```diff
@@ -2,10 +2,12 @@ import { Client, createClient, dedupExchange, errorExchange } from '@urql/core'
 import { executeExchange } from '@urql/exchange-execute'
 import { makeCacheExchange } from './urqlClient'
 import type * as stubData from './testStubCloudTypes'
+import type { navigationMenu } from './testNavigationMenu'
 import { clientTestSchema } from './clientTestSchema'
 
 export interface ClientTestContext {
   stubData: typeof stubData
+  navigationMenu: typeof navigationMenu
```
Comment (ImCesar) [inline packages/frontend-shared/src/graphql/testUrqlClient.ts:10]: The remaining check-ts failures are from app and wizard missing from this context. I wasn't sure if we wanted to create them like this or if we had a different plan like the stubdata.

**Review by ImCesar — COMMENTED** [review]
I fixed some of the build failures we were seeing. Feel free to change whatever and merge in when you're happy with it.

File: packages/frontend-shared/src/graphql/testUrqlClient.ts (hunk 1 above)
Comment (JessicaSachs) [reply packages/frontend-shared/src/graphql/testUrqlClient.ts:10]: Yeah, go ahead and create them there.

**Review by lmiller1990 — APPROVED** [review]

**Comment by lmiller1990** [comment]
Nice job, let's merge this and keep working on getting the decorators removal branch passing.

**Pull request merged**